### PR TITLE
Hack#56

### PR DESCRIPTION
**Pull request by aexshafii**
!!!IMPORTANT NOTE: feature not tested. 
Can't modify script within IE in LambdaTest as I do with overrides in Chrome

### CODE REVIEW

File: webflow-hacks/misc-hacks/56/56.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// when the DOM is ready
+document.addEventListener('DOMContentLoaded', function () {
+  // assign data attribute to constant
+  const POPUP_SELECTOR = '[fs-hacks-element="hack56-ie-popup"]';
+  // if browser is IE10 or IE11
+  if (/Trident\/|MSIE/.test(window.navigator.userAgent)) {
+    const POPUP = document.querySelector<HTMLElement>(POPUP_SELECTOR);
```
Comment (alexiglesias93) [inline webflow-hacks/misc-hacks/56/56.ts]: I think `POPUP` should be camelCase, not uppercase

**Review by alexiglesias93 — COMMENTED** [review]
I believe this hack can be CSS-only

**Comment by alexiglesias93** [comment]
I think this hack could be set without JavaScript, just CSS:

```html
<!--[if IE]>
  <style>
    [fs-hacks-element="upgrade-browser-popup"] {
      display: block;
    }
  </style>
<![endif]-->

<style>
  @media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
    [fs-hacks-element="upgrade-browser-popup"] {
      display: block;
    }
  }
</style>
```

**Comment by alexiglesias93** [comment]
Also, about testing:
the hacks are so small that you can just copy/paste them as inline JavaScript in a Webflow project, so you can test them without having issues with localhost/overrides in LambdaTest.

**Comment by aexshafii** [comment]
ready for review

**Review by alexiglesias93 — APPROVED** [review]

**Pull request merged**